dotnet/src/Connectors/Connectors.UnitTests/OpenAI/ChatCompletion/OpenAIChatCompletionServiceTests.cs

```diff
@@ -323,6 +323,212 @@ public async Task GetChatMessageContentsWithChatMessageContentItemCollectionAndS
         Assert.Equal("image_url", contentItems[1].GetProperty("type").GetString());
     }
 
+    [Fact]
+    public async Task FunctionCallsShouldBePropagatedToCallersViaChatMessageItemsOfTypeFunctionCallContentAsync()
```

Comment:
`FunctionCallRequestContent`? 
```suggestion
    public async Task FunctionCallsShouldBePropagatedToCallersViaChatMessageItemsOfTypeFunctionCallRequestContentAsync()
```